vendor/assets/javascripts/checkbox_modal.js

```diff
@@ -0,0 +1,26 @@
+// allow a cancel button to work properly on checkbox modal
+Cmsify.CheckboxModal = function(el) {
+  this.$el = $(el);
+  this.$el.addClass('uk-modal');
+  this.checked = {};
+  this.modal = UIkit.modal(el);
+  this.modal.on('show.uk.modal', function() {
+    this.storeCheckboxes();
+  }.bind(this));
+  this.$el.find('.js-cancel-' + this.$el.attr('id')).each(function(index, cancel) {
+    $(cancel).on('click', this.loadCheckboxes.bind(this)); 
```

Comment:
this should work because pressing save should submit the form rather than triggering ```hide.uk.modal``` like it normally does